editor/progress_dialog.cpp

```diff
@@ -147,6 +147,12 @@ void ProgressDialog::_popup() {
 	main->set_offset(SIDE_BOTTOM, -style->get_margin(SIDE_BOTTOM));
 
 	if (!is_inside_tree()) {
+		for (Window *window : host_windows) {
+			if (window->has_focus()) {
+				popup_exclusive_centered(window, ms);
```

Comment:
shouldn't this return after `popup_exclusive_centered` was called?

Reply:
Correct, even if it seems to work without.